notification/src/main/java/com/amazon/opendistroforelasticsearch/alerting/destination/client/DestinationHttpClient.java

```diff
@@ -54,6 +54,15 @@
     private static final int TIMEOUT_MILLISECONDS = (int) TimeValue.timeValueSeconds(5).millis();
     private static final int SOCKET_TIMEOUT_MILLISECONDS = (int)TimeValue.timeValueSeconds(50).millis();
 
+  	/**
+  	 * all valid response status
+  	 */
+  	private static final Set<Integer> VALID_RESPONSE_STATUS = new HashSet<>(
```

Comment:
Np: Should we call SUCCESS_RESPONSE_STATUS?  To me all the http codes are valid(1XX, 3XX, 4XX)